### PR TITLE
Need feedback -- Joshua Preciado

### PR DESCRIPTION
**Pull request by joshuapreciado**
image

### CODE REVIEW

File: dom-manipulation/main.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+var numberOfClick = 0;
+var buttonishot = document.querySelector('.hot-button');
+var clickcounter = document.querySelector('.click-count');
+function handleClick(event) {
+  numberOfClick++;
+  console.log(numberOfClick);
+  clickcounter.textContent;
+  if (numberOfClick.className < 4) {
```
Comment (BrettAlbright) [inline dom-manipulation/main.js]: number of click is a number, so it doesn't have a classname, and what you should be checking is the number of clicks, that is a numeric value which indicates the number of times clicked

File: dom-manipulation/main.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+var numberOfClick = 0;
+var buttonishot = document.querySelector('.hot-button');
```
Comment (BrettAlbright) [inline dom-manipulation/main.js]: all variables should be written in camelCase, buttonIsHot not buttonishot

File: dom-manipulation/main.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+var clickcounter = document.querySelector('.click-count');
+function handleClick(event) {
+  numberOfClick++;
+  console.log(numberOfClick);
+  clickcounter.textContent;
+  if (numberOfClick.className < 4) {
+    numberOfClick.className = "cold"
+  } else if (numberOfClick.className < 7) {
+    numberOfClick.className = "cool"
```
Comment (BrettAlbright) [inline dom-manipulation/main.js]: you need to update the classname of the hot-button element, as that is the one which needs to change color

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
feedback given

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**